### PR TITLE
Add a changelog entry for #26687 [ci skip]

### PR DESCRIPTION
**Pull request by kamipo**

### CODE REVIEW

File: activerecord/CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,13 @@
 
     *Prathamesh Sonpatki*
 
+*   Support index length and order options using both string and symbol
+    column names.
+
+    Fixes #27243.
+
+    *Ryuta Kamizono*
+
```
Comment (eileencodes) [inline activerecord/CHANGELOG.md]: Can you move this to the top of the file? Thanks!

Comment (kamipo) [inline activerecord/CHANGELOG.md]: #26687 (5ff0d828f1f39b894a4b6752bddc5b3b96a61ec1) was merged between #26266 (d7f55e987849c5a17f9d3152abd4fbacac08a509) and #26529 (6e948f480118bceb66b72a53874b800fbda05327).
Should we move the entry to the top of the file?

Comment (eileencodes) [inline activerecord/CHANGELOG.md]: Yes because it doesn't get added in order of commits but rather at the top of the file to avoid conflicts.

Comment (kamipo) [inline activerecord/CHANGELOG.md]: Okay I moved the entry to the top of the file!

Comment (eileencodes) [inline activerecord/CHANGELOG.md]: Thanks!

**Pull request merged**